schemas/2018-05-01/Microsoft.Maps.json

```diff
@@ -0,0 +1,81 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-05-01/Microsoft.Maps.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.Maps",
+  "description": "Microsoft Maps Resource Types",
+  "resourceDefinitions": {
+    "accounts": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the Maps Account."
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Maps/accounts"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2018-05-01"
+          ]
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource."
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Gets or sets a list of key value pairs that describe the resource. These tags can be used in viewing and grouping this resource (across resource groups). A maximum of 15 tags can be provided for a resource. Each tag must have a key no greater than 128 characters and value no greater than 256 characters."
+        },
+        "sku": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/Sku"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "The SKU of this account."
+        }
+      },
+      "required": [
+        "name",
```

Comment:
>name [](start = 9, length = 4)

don't you have a property bag for this resource? Most resources have a property called "properties".

Reply:
This is our current template that we currently accept, and is generated from our REST API specs.
We pass in an empty list for properties.

```json
{
    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
    "contentVersion": "1.0.0.0",
    "parameters": {
        "name": {
            "type": "string"
        },
        "pricingSpec": {
            "type": "string"
        }
    },
    "resources": [
        {
            "apiVersion": "2018-05-01",
            "name": "[parameters('name')]",
            "location": "global",
            "type": "Microsoft.Maps/accounts",
            "properties": {},
            "sku": {
                "name": "[parameters('pricingSpec')]"
            }
        }
    ]
}
```